tests/testproject/TestProject.AppHost/TestProgram.cs

```diff
@@ -120,8 +121,9 @@ public class TestProgram : IDisposable
             }
             if (!resourcesToSkip.HasFlag(TestResourceNames.mongodb))
             {
+                var mongoPassword = builder.AddParameter("mongodb-password");
                 var mongoDbName = "mymongodb";
-                var mongodb = AppBuilder.AddMongoDB("mongodb")
+                var mongodb = AppBuilder.AddMongoDB("mongodb", password: mongoPassword)
```

Comment:
Existing tests shouldn't be modified. They should stay working after your change.

Reply:
Resolved by removing the Uri.EscapeDataString